src/index.ts

```diff
@@ -3,10 +3,12 @@ process.env['AWS_NODEJS_CONNECTION_REUSE_ENABLED'] = '1';
 
 import { run } from 'cmd-ts';
 
+import { CliInfo } from './cli.info.ts';
 import { cmd } from './commands/index.ts';
 import { logger } from './log.ts';
 
 const startTime = performance.now();
+logger.info({ package: CliInfo }, 'Cli:Info');
```

Comment:
random thought, do we also want to log `process.argv.slice(2)` (aka all the arguments?)

Reply:
I committed a change.
The arguments would be logged twice (but not identical way) if the command actually starts:
```shell
[2025-05-22T03:56:28.494Z] INFO Cli:Info id=01JVV259SX2JKR7PYR116GJKGV package={ package=@linzjs/argo-tasks version=TEST } args={ 0=stac-setup 1=--gsd 2=0.3 3=--region 4=new-zealand 5=--geospatial-category 6=dem }
[2025-05-22T03:56:28.499Z] INFO Cli:Start id=01JVV259SX2JKR7PYR116GJKGV cli=stac-setup args={ verbose=false gsd=0.3 region=new-zealand geospatialCategory=dem output=file:///tmp/stac-setup/ } isArgo=false
```

I am ok with that. Could maybe rename `args` to something else like `userInputs`?

Reply:
How about a separate args logged at trace level, although I'm not sure we adjust the logger to trace until after the run, unless we have some sort of `TRACE=1` env var.

Either way maybe Im over thinking it and its too much effort, what are your thoughts?